modules/react/tabs/stories/Tabs.stories.mdx

```diff
@@ -22,80 +33,67 @@ yarn add @workday/canvas-kit-react
+`Tabs.Item` and `Tabs.Panel` subcomponents.
 
 <ExampleCodeBlock code={NamedKeys} />
 
-#### Right-to-Left
+### Right-to-Left
 
 `Tabs` supports right-to-left through the theme of the `CanvasProvider`.
 
 <ExampleCodeBlock code={RightToLeft} />
 
-#### Disabled Tab
+### Disabled Tab
 
-A `Tab.Item` can be disabled by passing `disabled={true}`.
+Set the `disabled` prop of a `Tabs.Item` to `true` to disable it.
 
 <ExampleCodeBlock code={DisabledTab} />
 
-#### Alternative Tab Stop
+### Alternative Tab Stop
 
 By default, tab panels are focusable for accessibility. If the contents of a tab panel have a
-focusable element, you may disable this default behavior by setting `tabIndex` to `undefined`. This
-example has a tab panel with a focusable button in the tab panel.
+focusable element, you may disable this default behavior by setting the `tabIndex` prop of
+`Tabs.Panel` to `undefined`. This example has a tab panel with a focusable button.
 
 <ExampleCodeBlock code={AlternativeTabStop} />
 
-### Advanced Usage
-
-Below are some examples for more advanced / custom use cases. We expect most people won't need to
-read this section, but if you're needing to go beyond basic examples or are curious, feel free to
-explore this section. While these examples are not exhaustive, they provide additional insight into
-what's possible.
-
-#### Hoisted Model
+### Single Tab Panel
 
-The `Tabs` component takes in an optional `model` property. If not defined, it will create and use
-its own. Hoisting the model allows access to state and events outside the `Tabs` component family.
-You can create a `TabsModel` using the `useTabsModel` hook and configure the model with the same
-configuration that the `Tabs` component receives. If you pass in a configured model, configuration
-properties passed to the `Tabs` component will be ignored.
+The compound component pattern allows for advanced composition. For example, `Tabs` can be composed
+to have only a single `Tabs.Panel` using attribute overrides and callbacks. More information about
+attributes and callbacks can be found in the prop tables below for each subcomponent.
 
-In this example, we'll set up external observation of the `TabsModel` state and set up an external
-button to trigger an event to change the currently active tab.
-
-<ExampleCodeBlock code={HoistedModel} />
-
-#### Single Tab Panel
-
-The compound component patterns allows advanced composition. For example, the tabs can be composed
-to have only a single tab panel using attribute overrides and callbacks. More info about attributes
-and callbacks are in property tables below for each subcomponent. In this example, we'll use a
-Hoisted model and the `activeTab` property of the state to show content of our `contents` object.
+In this example, we use a hoisted model and the `activeTab` property of the state to show content
+from the `contents` object.
 
 <ExampleCodeBlock code={SinglePanel} />
 
-#### Dynamic Tabs
+### Dynamic Tabs
```

Comment:
I was a little confused by the `Dynamic Tabs` example. I thought I could move Add Tab above the `tabs.map` that creates the `Tabs.Items` and still have Add Tab show up at the end of the tabs list (given the `index={tabs.length}`), but that resulted in Add Tab showing up at the beginning of the list.

Also, removing use of the `index` property from the example code doesn't seem to affect it. Am I misunderstanding how `index` is supposed to work?

Reply:
Index is used to inform the model where in the list each tab should go. This example always adds a new tab at the end. If you add a tab at the middle, but didn't send an index, the rendering would be correct, but keyboard navigation will get messed up. What will happen is the tab will render at the middle of the list, but register with the model at the end of the list. So when you're focus is to the left of the new tab and you hit the right key, it will skip over the new tab. When you're focus is on the last visible tab and you hit the right key it will jump to the new tab in the middle of the list.